components/helper_functions/actions/trigger-workflow/trigger-workflow.mjs

```diff
@@ -0,0 +1,49 @@
+import helperFunctions from "../../helper_functions.app.mjs";
+
+export default {
+  key: "helper_functions-trigger-workflow",
+  name: "Trigger Workflow",
+  description: "Invokes another workflow by its ID.",
+  version: "0.0.1",
+  type: "action",
+  props: {
+    helperFunctions,
+    workflowId: {
+      type: "string",
+      label: "Workflow ID",
+      description: "The ID of the workflow to trigger. Workflow IDs are formatted as `p_******` and you can find a workflow’s ID within the workflow builder URL.",
+    },
+    event: {
+      type: "object",
+      label: "Event",
+      description: "The event to be sent to the triggered workflow as the triggering event",
+      optional: true,
+    },
+    resultType: {
```

Comment:
I think we should just default to `all_exports` for now, and hide the prop. It could be used down the line if there are additional `result_type` added, so let's keep the `result_type` prop present, but hidden with a default setting.